userdetails.go

```diff
@@ -0,0 +1,13 @@
+// +build windows linux darwin
```

Comment:
Why limit this to just the three OSs? E.g. it will probably work on other BSDs just fine.

I get that this project probably only works on these three platforms, but if we really wanted to break the build for other OSs, I don't think this is the place to do it.

Also, you can just simplify the code if you remove the build tags.

Reply:
> Also, you can just simplify the code if you remove the build tags.

Per Slack chat, agree that because this is a tool it doesn't make sense to solve the "does this work on OS ABC" via build tags.